app_grim_reefer/boards/arm/grim_reefer/grim_reefer.dts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2023 Aaron Chan
```

Comment:
Maybe update or remove most of what you know won't be used. Also update the copyright 😉

Reply:
fixed in [be390f3](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/be390f371979b6511bdb8df2cc744ed8e4391bab)